include/openenclave/internal/switchless.h

```diff
@@ -12,10 +12,21 @@
 typedef struct _host_worker_thread_context
 {
     volatile oe_call_host_function_args_t* call_arg;
-    volatile bool is_stopping;
+    uint64_t is_stopping;
```

Comment:
Did you make this uint64_t so it would alignment reasons?

Reply:
Yes. I have now locked the entire layout using static-asserts.